spec/models/npq_application_spec.rb

```diff
@@ -34,7 +34,7 @@
     end
 
     context "when cohort_id is modified" do
-      let(:cohort) { create(:cohort) }
+      let(:cohort) { create(:cohort, :next) }
```

Comment:
Ah you encountered this too, I attempted to fix this in: https://github.com/DFE-Digital/early-careers-framework/pull/3155